docs/tests/t007.md

```diff
@@ -0,0 +1,106 @@
+# T007: subject group classes
+**Abstract:**  
+This documents contains the definitions for tests that concern the management of subject group classes. Study group classes are "groups of groups", that can be used to label and filter study subjects.  

```

Comment:
should "Study group classes" be "Study subject group classes"?

Reply:
It's a bit strange, but in the form it actually says Study groups. Maybe we should change that eventually in the interface. For now I will change this text and only use "subject".